From 1c6be22c5eeea9a7f83192153b282b6002b89923 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Mon, 12 Apr 2021 22:44:06 +0200
Subject: [PATCH] server/drivers/g15: Add support for the LCD on Logitech Z-10
 speakers

The LCD on Logitech Z-10 speakers uses the exact same protocol as on
the g15 keyboards, add the USB-id + HID-descriptor header of the
USB-intf. for the LCD.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 server/drivers/g15.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/server/drivers/g15.c b/server/drivers/g15.c
index 154e253..48f5fe5 100644
--- a/server/drivers/g15.c
+++ b/server/drivers/g15.c
@@ -90,6 +90,10 @@ static const struct lib_hidraw_id hidraw_ids[] = {
 	{ { BUS_USB, 0x046d, 0xc22e },
 	  { 0x05, 0x0c, 0x09, 0x01, 0xa1, 0x01, 0x85, 0x02,
 	    0x15, 0x00, 0x25, 0x01, 0x75, 0x01, 0x95, 0x07 } },
+	/* Z-10 */
+	{ { BUS_USB, 0x046d, 0x0a07 },
+	  { 0x06, 0x00, 0xff, 0x09, 0x00, 0xa1, 0x01, 0x15,
+	    0x00, 0x26, 0xff, 0x00, 0x75, 0x08, 0x95, 0x08 } },
 	/* Terminator */
 	{}
 };
-- 
2.31.1

